bot/cogs/watchchannels/bigbrother.py

```diff
@@ -0,0 +1,118 @@
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def watched_command(self, ctx: Context, update_cache: bool = True) -> None:
+        """
+        Shows the users that are currently being monitored in BigBrother.
+
+        The optional kwarg `update_cache` can be used to update the user
+        cache using the API before listing the users.
+        """
+        await self.list_watched_users(ctx, update_cache)
+
+    @bigbrother_group.command(name='watch', aliases=('w',))
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def watch_command(self, ctx: Context, user: Union[User, proxy_user], *, reason: str) -> None:
+        """
+        Relay messages sent by the given `user` to the `#big-brother-logs` channel.
+
+        A `reason` for adding the user to BigBrother is required and will displayed
+        in the header when relaying messages of this user to the watchchannel.
+        """
+        if user.bot:
+            e = Embed(
+                description=f":x: **I'm sorry {ctx.author}, I'm afraid I can't do that. I only watch humans.**",
+                color=Color.red()
+            )
+            await ctx.send(embed=e)
+            return
+
+        if not await self.fetch_user_cache():
+            log.error("Failed to update user cache; can't watch user {user}")
+            return
+
+        if user.id in self.watched_users:
+            e = Embed(
+                description=":x: **The specified user is already being watched**",
+                color=Color.red()
+            )
+            await ctx.send(embed=e)
+            return
+
+        response = await post_infraction(
+            ctx, user, type='watch', reason=reason, hidden=True
+        )
+        if response is not None:
+            self.watched_users[user.id] = response
+            e = Embed(
+                description=f":white_check_mark: **Messages sent by {user} will now be relayed to BigBrother**",
+                color=Color.green()
+            )
+            await ctx.send(embed=e)
+            return
+
+    @bigbrother_group.command(name='unwatch', aliases=('uw',))
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def unwatch_command(self, ctx: Context, user: Union[User, proxy_user], *, reason: str) -> None:
+        """Stop relaying messages by the given `user`."""
+        active_watches = await self.bot.api_client.get(
+            self.api_endpoint,
+            params=ChainMap(
+                self.api_default_params,
+                {"user__id": str(user.id)}
+            )
+        )
+        if active_watches:
+            [infraction] = active_watches
+            await self.bot.api_client.patch(
+                f"{self.api_endpoint}/{infraction['id']}",
+                json={'active': False}
+            )
+            await post_infraction(
+                ctx, user, type='watch', reason=f"Unwatched: {reason}", hidden=True, active=False
```

Comment:
Another style thing where I think the arguments should be on separate lines. There's more instances of this but I'm not going to leave a comment on each one.

Reply:
I don't agree with this one, since this has all the arguments on a single line and this is actually the style that `black` would also produce.

Reply:
this will override the reason of the existing watch, I think we should append to it instead. i also agree on multiple lines, just because black would produce this style doesn't mean its good

Reply:
I'll change the style, although the argument wasn't "it's good because `black` does it". My personal preference is the way it was, that's why I coded it that way, and not the version that smears it out over multiple lines unnecessarily. The `black` part came from the idea of using `black` on the bot code base, which will change this back to the single line version anyway.

(I know the black idea is controversial, so it may never happen.)

Reply:
> this will override the reason of the existing watch, I think we should append to it instead.

No, it will add a new `Watch` infraction with the unwatch reason. That way, not only the unwatch reason is stored, but also the date and actor.